view/next-project/src/type/common.ts

```diff
@@ -232,3 +232,13 @@ export interface PurchaseReportView {
   reportUser: User;
   purchaseItems: PurchaseItem[];
 }
+
+// // Year(年度)
+export interface YearPeriods {
```

Comment:
1つの年度を示す型なのに複数形は違和感があります

```suggestion
export interface YearPeriod {
```

Reply:
修正しました [[fix]type名変更](https://github.com/NUTFes/FinanSu/pull/660/commits/0668b50cd8e3e3ab0a9718d1adb9bcbb8d3be4bd)